homeassistant/components/sensor/onewire.py

```diff
@@ -34,63 +50,54 @@
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Set up the one wire Sensors."""
     base_dir = config.get(CONF_MOUNT_DIR)
-    sensor_ids = []
-    device_files = []
+    devs = []
+    device_names = {}
+    if 'names' in config.keys():
```

Comment:
No need to add `.keys()` to test if a key exists.

Reply:
Will update.